R/apply_fmt_fn.R

```diff
@@ -54,6 +57,8 @@ apply_fmt_fn <- function(x) {
 #'
 #' @param x (`integer`, `string`, or `function`)\cr
 #'   a non-negative integer, string alias, or function
+#' @param stat the statistic that is to be formatted
+#' @param stat_name the name of the statistic that is to be formatted
```

Comment:
Add parameter types to documentation
```suggestion
#' @param stat (`numeric`)\cr the value of the statistic that is to be formatted
#' @param stat_name (`character`)\cr the name of the statistic that is to be formatted
```